apps/api/src/routes/applicationRouter.ts

```diff
@@ -284,57 +284,85 @@ applicationRouter.post('/reject', jsonParser, async (req, res) => {
 	}
 });
 
-// POST: Submit revisions
+/**
+ * TODO: NO current Auth rules implemented
+ */
 applicationRouter.post(
-	'/:applicationId/submit-revision',
-	jsonParser,
-	withParamsSchemaValidation(
-		collaboratorsListParamsSchema,
-		apiZodErrorMapping,
-		async (request: Request, response: Response) => {
-			const { applicationId } = request.params;
-
-			if (!applicationId || !isPositiveInteger(Number(applicationId))) {
-				response.status(400).json({
-					message: 'Invalid request. ApplicationId is required and must be a valid number.',
-					errors: 'MissingOrInvalidParameters',
-				});
-			}
+	'/file/ethics/:applicationId',
+	fileUploadValidation(async (req: Request<any, { file: formidable.File }>, res: Response) => {
+		const { applicationId } = req.params;
+
+		const { file } = req.body;
+
+		const id = parseInt(applicationId ? applicationId : '');
+		if (!isPositiveInteger(id)) {
+			res.status(400).send({ message: 'Invalid applicationId' });
+			return;
+		}
 
-			try {
-				const applicationIdNum = Number(applicationId);
-				const result = await submitRevision({ applicationId: applicationIdNum });
+		const result = await uploadEthicsFile({ applicationId: id, file });
 
-				if (result.success) {
-					response.status(200).send({
-						message: 'Application review submitted successfully.',
-						data: result.data,
+		if (result.success) {
+			res.status(200).send(result.data);
+			return;
+		} else {
+			const errorReturn = { message: result.message, errors: String(result.errors) };
+			res.status(500).send(errorReturn);
+			return;
+		}
+	}),
```

Comment:
I think you're missing a closing `)` here
(posted twice, first time was wrong line)